Dockerfile

```diff
@@ -123,6 +144,13 @@ COPY clients/ .
 ####################
 FROM frontend AS built_frontend
 
+# Copy the extracted version from the version_extractor stage
+COPY --from=version_extractor /version.txt /tmp/version.txt
+
+# Replace the placeholder version in next.config.js with the actual version
+RUN RELEASE_VERSION=$(cat /tmp/version.txt) && \
+    sed -i "s/__RELEASE_VERSION__/$RELEASE_VERSION/g" /fides/clients/privacy-center/next.config.js
```

Comment:
nit: This makes sense, but... I dunno, I think it's an important "build" command that I'd expect to happen via `npm run build-privacy-center` instead. So that build command could look for the existence of the `clients/version.txt` file and use it, and you wouldn't have to rely on `sed` as a peculiar side door